lib/MusicBrainz/Server/Controller/Role/EditListing.pm

```diff
@@ -4,6 +4,10 @@ use Moose::Role -traits => 'MooseX::MethodAttributes::Role::Meta::Role';
 use MusicBrainz::Server::Data::Utils qw( model_to_type load_everything_for_edits );
 use MusicBrainz::Server::Constants qw( :edit_status );
 
+MusicBrainz::Server::Controller->config(
+    paging_limit => 50,
+);
```

Comment:
This looks like it sets it "globally" (for all controllers) rather than on the specific package implementing this role.

Can we just pass `limit => 50` to the relevant `_load_paged` calls?

Reply:
I guess that would be reasonable, yeah. Done.